esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -67,30 +72,34 @@
             ROOTPATH_OBS4MIPS = {{ ROOTPATH_OBS4MIPS }}
             ROOTPATH_RAWOBS = {{ ROOTPATH_RAWOBS }}
 
+    # Bash variable substitution below replaces `--` with `/` to obtain path to
+    # recipe.
     [[process<fast>]]
         inherit = None, COMPUTE
         [[[environment]]]
             ROSE_TASK_APP = process
-            RECIPE_NAME = "recipe_${CYLC_TASK_PARAM_fast}.yml"
+            RECIPE_PATH = "${CYLC_TASK_PARAM_fast//--//}.yml"
 
     [[process<medium>]]
         inherit = None, COMPUTE
         [[[environment]]]
             ROSE_TASK_APP = process
-            RECIPE_NAME = "recipe_${CYLC_TASK_PARAM_medium}.yml"
+            RECIPE_PATH = "${CYLC_TASK_PARAM_medium//--//}.yml"
 
+    # Bash variable manipulation `##*--` below strips leading path from recipe
+    # parameter to obtain recipe name (=recipe basename minus yml extension).
```

Comment:
As discussed, would it be possible to extend this comment to provide a real life example, please? (What confused me was the fact that `${CYLC_TASK_PARAM_fast}` doesn't have a yml extension, and also that the output directory created by ESMValTool is based on the recipe name and not on any directories that are specified relative to the `esmvaltool/recipes/` directory) 😊

Reply:
Good point. How is this: [ca06201](https://github.com/ESMValGroup/ESMValTool/pull/3567/commits/ca06201df4b3b814a4b466bdc70e34badf55d824)